service/controller/v3/network/network.go

```diff
@@ -0,0 +1,74 @@
+package network
+
+import (
+	"context"
+	"net"
+
+	"github.com/giantswarm/ipam"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/azure-operator/service/controller/v3/key"
+)
+
+const (
+	masterSubnetMask = 24
+	workerSubnetMask = 24
+	vpnSubnetMask    = 24
+
+	ipv4MaskSize = 32
+)
+
+// ComputeFromCR computes subnets using network found in CR.
+func ComputeFromCR(ctx context.Context, obj interface{}) (*Subnets, error) {
```

Comment:
I would just call it `FromCR`. Package name indicates it will be `network`

Reply:
Any idea to rename `Compute` then ?

Reply:
I see. Given https://github.com/giantswarm/azure-operator/pull/277/files#r199463805 this name makes sense.